scripts/generateUsersActivityReportCsv.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * NODE_ENV=beta node ./scripts/generateUsersActivityReportCsv.js
+ */
+
+const { writeFileSync } = require('fs');
+const config = require('config');
+const { Parser } = require('json2csv');
+const mongoose = require('mongoose');
+
+const appFactory = () => {
+  const data = {};
+  return {
+    get(key) {
+      return data[key];
+    },
+    set(key, val) {
+      data[key] = val;
+    },
+  };
+};
+
+const app = appFactory();
+app.set('mongooseClient', mongoose);
+const aggregteQuery = [
+  {
+    $match: {},
```

Comment:
You can change it to giverId not equal zero

Reply:
For instance this user just has signed in and didn't create a giver in smart contract

Reply:
Are you sure ?
for example check below user it has a lot of activity but his giverId is zero 
<img width="1247" alt="Screen Shot 1400-01-27 at 08 52 26" src="https://user-images.githubusercontent.com/9850545/114971324-5812b680-9e91-11eb-88d4-502b3d78797b.png">


https://feathers.beta.giveth.io/users/0x51e37e3716c0ED418a90e519CeEE395d8f6deCB9

But I changed to check the users that have `giverId` field and it has better result,
but if you think `giverId:0` still is ok tell me to add it